src/components/modules/epics/ContributionsEpicReminderSignedOut.tsx

```diff
@@ -0,0 +1,241 @@
+
+// --- Styles --- //
+
+const rootStyles = css`
+    position: relative;
+
+    * {
+        box-sizing: border-box;
+    }
+`;
+
+const closeButtonStyles = css`
+    width: 30px;
+    height: 30px;
+    cursor: pointer;
+    position: absolute;
+    top: 20px;
+    right: 0;
+    background: none;
+    border: none;
+    padding: 0;
+`;
+
+const lineWrapperStyles = css`
+    margin: ${space[3]}px auto;
+`;
+
+const containerStyles = css`
+    padding: 0 ${space[1]}px;
+`;
+
+const remindHeading = css`
+    ${headline.xxsmall({ fontWeight: 'bold' })};
+    margin: 0 ${space[5]}px ${space[2]}px 0;
+`;
+
+const successTextStyles = css`
+    margin: 0 auto ${space[2]}px;
+    ${body.medium()};
+`;
+
+const linkStyles = css`
+    color: ${palette.neutral[7]};
+`;
+
+const formWrapper = css`
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+
+    ${from.tablet} {
+        flex-direction: row;
+        align-items: flex-end;
+        justify-content: flex-start;
+    }
+`;
+
+const inputWrapper = css`
+    width: 100%;
+    margin-bottom: ${space[2]}px;
+    flex-grow: 1;
+
+    ${from.tablet} {
+        width: auto;
+        margin-right: ${space[2]}px;
+        margin-bottom: 0;
+    }
+`;
+
+const formTextStyles = css`
+    ${textSans.small()};
+    font-style: italic;
+    margin-top: ${space[1]}px;
+`;
+
+const errorTextStyles = css`
+    ${textSans.small({ fontWeight: 'bold' })};
+    color: ${palette.error[400]};
+    font-style: italic;
+    margin-top: ${space[1]}px;
+    margin-bottom: 0;
+`;
+
+const getCustomSubmitStyles = (isDisabled: boolean): SerializedStyles | undefined => {
+    if (isDisabled) {
+        // Unfortunately these overrides need !important as otherwise they'll lose
+        // the specificity war against the default Source styles.
+        return css`
+            pointer-events: none;
+            color: ${palette.neutral[60]} !important;
+            background-color: ${palette.neutral[93]} !important;
+            border: 1px solid ${palette.neutral[86]} !important;
+        `;
+    }
+
+    return undefined;
+};
+
+// --- Types --- //
+
+export interface ContributionsEpicReminderSignedOutProps {
+    reminderLabel: string;
+    reminderStatus: ReminderStatus;
+    onSetReminderClick: (email: string) => void;
+    onCloseReminderClick: () => void;
+}
+
+// --- Components --- //
+
+export const ContributionsEpicReminderSignedOut: React.FC<ContributionsEpicReminderSignedOutProps> = ({
+    reminderLabel,
+    reminderStatus,
+    onSetReminderClick,
+    onCloseReminderClick,
+}: ContributionsEpicReminderSignedOutProps) => {
+    const [isDirty, setIsDirty] = useState(false);
+    const [emailAddress, setEmailAddress] = useState('');
+
+    const updateEmail = (e: React.ChangeEvent<HTMLInputElement>) => {
+        setEmailAddress(e.target.value);
+        if (isDirty) {
+            setIsDirty(false);
+        }
+    };
```

Comment:
This is the one place that I've changed the UX of the original form in the refactor. `isDirty` would maybe be better named `shouldShowErrorMessage`. It's set to `true` when submitting an invalid email address. The change I've made is to now set is back to `false` when the user starts typing again. This way, as soon as a user starts typing again, it will remove the error message. The current form will continue to display it until they type in a valid email address.

Reply:
it's a difficult one to name! `shouldShowInputError`?

Reply:
the change of behaviour sounds sensible though